From c4120b62b5592a6634a1a8ad6c45638e1ac8be8e Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Tue, 25 Dec 2018 16:19:51 -0500
Subject: [PATCH] disable seed-based field trials

This also now disablse setting up client-side field trials as well
---
 .../variations/service/variations_field_trial_creator.cc      | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/components/variations/service/variations_field_trial_creator.cc b/components/variations/service/variations_field_trial_creator.cc
index 63b4f52474828..0ef06b356a2f7 100644
--- a/components/variations/service/variations_field_trial_creator.cc
+++ b/components/variations/service/variations_field_trial_creator.cc
@@ -311,12 +311,16 @@ bool VariationsFieldTrialCreator::SetUpFieldTrials(
 
   bool used_seed = false;
   if (!used_testing_config) {
+#if defined(FIELDTRIAL_SEED_ENABLED)
     used_seed = CreateTrialsFromSeed(*entropy_providers, feature_list.get(),
                                      safe_seed_manager);
+#endif // BUILDFLAG(FIELDTRIAL_SEED_ENABLED)
   }
 
+#if defined(FIELDTRIAL_SEED_ENABLED)
   platform_field_trials->SetUpClientSideFieldTrials(
       used_seed, *entropy_providers, feature_list.get());
+#endif // BUILDFLAG(FIELDTRIAL_SEED_ENABLED)
 
   base::FeatureList::SetInstance(std::move(feature_list));
 
